src/dataframe/io.jl

```diff
@@ -801,7 +801,7 @@ function readtable(io::IO,
                    separator::Char = ',',
                    quotemark::Vector{Char} = ['"'],
                    decimal::Char = '.',
-                   nastrings::Vector = ["", "NA"],
+                   nastrings::Vector = ["", "NULL", "NA"],
```

Comment:
I think it's worth noting that Hive prints out `NULL` by default and that R uses `NA` by default. If people are unhappy with the defaults, I think the only defensible solution is to have no default and require people to specify things manually.